shared_python/Chapters.py

```diff
@@ -142,6 +143,8 @@ def populate_chapters(self, folder=None, extensions=None):
         else:
             for _, chapter_path in file_paths.items():
                 path = chapter_path.replace(self.args.chapters_path, "")[1:]
+                if os.sep == "\\":  # if this script is run on windows
+                    path = path.replace("\\", "/")
```

Comment:
I thought the problem you had was the reverse, that the existing paths contained `/` but you needed to change them to `\\` if you're on Windows - or am I misunderstanding what this is doing?

Reply:
It's easier to see in the file but this path variable is used in the query that updates the chapter with the text. On windows, this path variable has `\\` but the url in the chapters table will only have `/` (if present at all). That's why changing the urls in the chapters table to use `\\` worked, but this part avoids the need to update the data. Sorry if I was unclear about the issue before!